packages/public/server/src/module/Authentication/src/Service/AuthenticationService.php

```diff
@@ -62,6 +63,11 @@ class AuthenticationService extends ZendAuthenticationService implements
      */
     protected $cookiePath = '/';
 
+    /**
+     * @var UserInterface
+     */
+    protected $identity = null;
+
```

Comment:
As I said above: we probably don't want to use this `AuthenticationService` for that. Please create a new `AuthenticationService` that is only intended for the API where we have a better way to make sure that we don't introduce side effects. If feasible, we don't want to use an `AuthenticationService` at all (since this uses `Zend\Authentication\AuthenticationService` under the hood, where we again have no control over what's happening).